packages/@aws-cdk/cdk/test/cloudformation/test.resource.ts

```diff
@@ -349,6 +349,37 @@ export = {
     test.done();
   },
 
+  'TaggableResource': {
+    'TaggableResource with tags from an L1 constructor overrides tag manager'(test: Test) {
+      const stack = new Stack();
+      const tagResource = new TaggableResource(stack, 'TaggableGuy', {
+        type: 'SomeTaggableResource',
+        properties: {
+          tags: [ {key: 'tagKey', value: 'tagValue'} ],
+        },
+      });
+      tagResource.tags.setTag('tagKey', 'notTagValue');
```

Comment:
I have to say that my expectation was that `setTag` post initialization will override the one provided when I initialized (or crash or allow me to decide if I want to override or now)

Reply:
going to change this once the ctor has the initial tags.